test/chunking-form/samples/chunking-compact/_expected/amd/chunk-1e9855f7.js

```diff
@@ -0,0 +1,6 @@
+define(['exports'],function(exports){'use strict';function fn () {
+  console.log('lib2 fn');
+}function fn$1 () {
+  fn();
+  console.log('dep2 fn');
+}exports.a=fn$1;});
```

Comment:
and again we should consider adding semicolons even if it means adding empty statements in some situations. Alternatively, line-breaks might work as well but would not solve issues where the succeeding code starts with an opening bracket.